packages/connectors/connector-aliyun-dm/src/utils.ts

```diff
@@ -41,11 +49,13 @@ export const request = async (
   parameters: PublicParameters & Record<string, string>,
   accessKeySecret: string
 ) => {
-  const finalParameters: Record<string, string> = {
-    ...parameters,
-    SignatureNonce: String(Math.random()),
-    Timestamp: new Date().toISOString(),
-  };
+  const finalParameters = Object.fromEntries<string>(
+    Object.entries<string | undefined>({
+      ...parameters,
+      SignatureNonce: generateStandardId(),
```

Comment:
anything wrong with `String(Math.random())`?

Reply:
It can work properly, but the `Math.random()` is generating a float between 0-1, hence `String(Math.random())` produces strings like "0.1267831" (which look not elegant).

Reply:
how about add `nanoid` as a dependency instead

Reply:
Can use `crypto.randomUUID()` instead of introducing additional dependencies.